File: python/udemy/4-estrutura-condicional/dardo.py
# No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir. Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual foi a maior.

print('Digite a distância de cada tentativa.')
t1 = float(input('Tentativa 1: '))
t2 = float(input('Tentativa 2: '))
t3 = float(input('Tentativa 3: '))

if t1 > t2 and t1 > t3:
    maior = t1
elif t2 >= t1 and t2 >= t3:
    maior = t2
else:
    maior = t3

print(f'Maior distância: {maior}.')

File: c/udemy/4-condicional/dardo.c
// No arremesso de dardo, o atleta tem três chances para lançar o dardo à maior distância que conseguir. Você deve criar um programa para, dadas as medidas das três tentativas de lançamento, informar qual foi a maior

#include <stdio.h>

int main(){

    double d1, d2, d3, maior;

    printf("Digite as 3 tentativas:\n");
    scanf("%lf %lf %lf", &d1, &d2, &d3);

    if (d1 > d2 && d1 > d3){
        maior = d1;
    }
    else if (d2 >= d3 && d2 >= d1){
        maior = d2;
    }
    else {
        maior = d3;
    }

    printf("Maior tentativa = %.2lf\n", maior);

    return 0;
}